carbonmark/theme/globals.css

```diff
@@ -18,3 +18,7 @@ a:visited {
 [aria-modal="true"] ::-webkit-scrollbar {
   width: 1.6rem;
 }
+
+.mapboxgl-canvas {
+  width: 100% !important;
```

Comment:
Again on the `!important` can we avoid it?

Reply:
So for this one there was a layout shift happening on the map view - currently visible in production also, where the map isn't taking the full width and there is a grey background showing on the right hand side of the page. The mapbox library looks to apply these styles inline, so this was unavoidable in this scenario to overwrite the inline styles.